### DISCUSSION TITLE
Amplitude value

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#1345] [General]
Hello,
In the Parameters of an organ running, does the value of the Amplitude has to be 100 or upper ?

I tried to put 0, because I felt my organ too loud, and the result was sound off ..

**Comment by kerkovits** [comment]
As far as I know, 0 means no sound, 100 means original amplitude, any number in between changes the volume proportionally.

**Reply by rousseldenis** [reply]
@Jean-Gabriel1982 You have the explanation in the Help > Organ Settings (the second one) > Amplitude.

I hope this will help you

**Reply by Jean-Gabriel1982** [reply]
Hello, many thanks for your answers !
I will look at the manual ;-)
